verilog/analysis/verilog_excerpt_parse.cc

```diff
@@ -69,6 +69,12 @@ static std::unique_ptr<VerilogAnalyzer> AnalyzeVerilogConstruct(
   return analyzer_ptr;  // Let caller check analyzer_ptr's status.
 }
 
+std::unique_ptr<VerilogAnalyzer> AnalyzeVerilogProperty(
```

Comment:
Do you want to register this with `AnalyzeVerilogWithMode()`, below?  If so, add one test in verilog_analyzer_test.cc.

Reply:
Done